src/tooltip.tsx

```diff
@@ -1,20 +1,30 @@
 import * as React from 'react';
 import classnames from 'classnames';
+import {CSSTransition} from 'react-transition-group';
 import {useAriaId, useScreenSize} from './hooks';
 import {Portal} from './portal';
 import Overlay from './overlay';
+import {Text2} from './text';
 import * as key from './utils/key-codes';
 import {createUseStyles} from './jss';
+import Stack from './stack';
 
 const defaultPositionDesktop = 'bottom';
 const defaultPositionMobile = 'top';
 const arrowSize = 12;
-const distanceToTarget = 8 + arrowSize;
+const distanceToTarget = 4 + arrowSize;
 const marginLeftRightMobile = 16;
 const defaultWidthDesktop = 340;
 const arrowWrapperWidth = arrowSize * 2;
 const arrowWrapperHeight = arrowSize;
 
+const transitionDurationMs = 500;
+const animationMovement = 12;
+const animationTiming = 'cubic-bezier(0.215, 0.61, 0.355, 1)';
+const defaultShowTooltipDelayMs = 2000;
```

Comment:
too much, I'd say something like 500ms should be fine

Reply:
we can change it to 500ms

But when you hover some elements for example, PR Chip in this page, the delay is more than 500ms, I think is a native delay, just comment
![image](https://user-images.githubusercontent.com/6722153/195676570-fa76102b-2576-4626-960f-2fc12692eaa8.png)